pkg/api/alert_client.go

```diff
@@ -23,6 +71,97 @@ type AlertSettingsConditionPayload struct {
 	Channels    []string `json:"channels" yaml:"channels"`
 }
 
+func constructQueryString(page int, pageSize int, sortField, sortOrder, templateName, channelName string) (query string) {
+	path := fmt.Sprintf("%s?pageSize=%d", pkg.AlertChannelsPath, pageSize)
+	if page != 0 {
```

Comment:
there is a nice library (not for now but for the future) which is called `validator.v10` which validates the required paremeters as struct.